### PR TITLE
Edit A records

### LINKED ISSUE #1141: Implement add / edit A/AAAA record

### PR DESCRIPTION
**Pull request by eatonphil**
Closes #1141

### CODE REVIEW

**Comment by abemassry** [comment]
Have a question about AAAA records, can you provide a screen shot?

File: test/dnsmanager/components/EditARecord.spec.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        dispatch={dispatch}
+        zone={currentZone}
+        close={close}
+      />
+    );
+
+    const changeInput = (name, value) =>
+      page.instance().setState({ [name]: value });
+
+    changeInput('ip', '1.1.1.8');
```
Comment (abemassry) [inline test/dnsmanager/components/EditARecord.spec.js]: maybe something like 2001:0db8:85a3:0000:0000:EA75:1337:BEEF here

File: test/dnsmanager/components/EditARecord.spec.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+    expect(dispatch.callCount).to.equal(1);
+    expect(close.callCount).to.equal(1);
+
+    const fn = dispatch.firstCall.args[0];
+    await expectRequest(
+      fn, `/dns/zones/${currentZone.id}/records/`, undefined, undefined, {
+        method: 'POST',
+        body: {
+          target: '1.1.1.8',
```
Comment (abemassry) [inline test/dnsmanager/components/EditARecord.spec.js]: and here something like 2001:0db8:85a3:0000:0000:EA75:1337:BEEF

**Comment by mparke** [comment]
can't submit:

screen shot 2017-02-21 at 10 10 38 am

**Comment by mparke** [comment]
when opening the modal

Warning: Failed prop type: Invalid prop `disabled` of type `number` supplied to `Select`, expected `boolean`.

**Comment by eatonphil** [comment]
@mparke fixed:

screen shot 2017-02-23 at 10 22 50 am

**Comment by eatonphil** [comment]
thanks!

**Pull request merged**